docs/UserGuide.md

```diff
@@ -5,62 +5,94 @@ title: User Guide
 
 AgentAssist is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AgentAssist can get your sales tasks done faster than traditional GUI apps.
 
-* Table of Contents
-{:toc}
+- Table of Contents
+  {:toc}
 
---------------------------------------------------------------------------------------------------------------------
+---
 
 ## Quick start
 
-1. Ensure you have Java `17` or above installed in your Computer.
+### 1. Install Java
+
+Make sure you have **Java 17 or above** installed on your computer. You may skip this step if you already have it installed.
+
+Installing Java:
+
+- Download Java [here](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html). Follow the instructions on that page to install Java.
+
+### 2. Download the AgentAssist application
+
+Go to this [link](https://github.com/AY2425S1-CS2103T-T14-4/tp/releases) and download the latest version of the `.jar` file.
+
+### 3. Choose a Folder
+
+Find or create a folder on your computer where you want to store the AgentAssist information. <br>
+Move the `.jar` file you just downloaded into this folder.
+
+### 4. Open a Command Terminal
+
+Now, open a command terminal:
+
+- On Windows, press the **Windows Key** and type **`Command Prompt`** in the search bar. Click on the **Command Prompt** application to open it.
+- On macOS, press **Cmd + Space**, type **`Terminal`** into the search bar, and press **Enter**.
+- On Linux, open your Terminal application.
+
+### 5. Run the applicaton
+
+Navigate your terminal to the folder where you saved the AgentAssist application:
 
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
+- Type **`cd `** (with a space after it), then drag and drop the folder where you placed the `.jar` file into the terminal window. Press Enter.<br>
```

Comment:
maybe add an example? e.g. `cd samplefolder`

Reply:
Done! Give it a check.